external/fv3fit/tests/training/test_main.py

```diff
@@ -387,6 +392,8 @@ def cli_main(args: MainArgs):
         + validation_args
         + local_download_args
     )
+    # if you need pdb support, temporarily replace the check_call above with this:
+    # fv3fit.train.main(args)
```

Comment:
Could delete this, but I found it handy, and it took a little while to figure out what to replace it with here.